### ISSUE TITLE
Add new attributes

### ISSUE THREAD

**Issue by Dhruvkanani** [Chrisvin/FlipTabs#3]
I want to make the border of fliptabs as per the value I enter. I also want the text inside the fliptabs to be customisable like text size, text style, fontfamily,etc.  There is also the problem in size as whenever I try to make it small it cuts from all sides. It will be a pleasure if you solve these problems. Thank You.

**Comment by Chrisvin** [comment]
Resolved in version **v1.3** 

Border width can be set programmatically using `setBorderWidth` or as an xml attribute `app:borderWidth` .
Text can be customized by getting the TextViews & applying necessary changes programmatically.
Eg:
```
//Modify properties of the textviews used in FlipTab
fliptab.getTextViews().forEach {
    it.setTypeface(it.typeface, Typeface.ITALIC)
}
//Or modify a specific textview
fliptab.getSelectedTextView().apply {
    setTypeface(typeface, Typeface.BOLD)
}
// Similarly, fliptab.getLeftTextView() & fliptab.getRightTextView() can be used
```
Have identified and fixed a size-related issue.

**Comment by Chrisvin** [comment]
If issue still persists, feel free to reopen the issue.
